src/main.cpp

```diff
@@ -102,7 +107,15 @@ int main(int argc, char** argv)
 
     MainWindow mainWindow;
     app.setMainWindow(&mainWindow);
-    
+
+    QObject::connect(&app, &Application::anotherInstanceStarted,
+                    [&]() {
+                        mainWindow.ensurePolished();
+                        mainWindow.setWindowState(mainWindow.windowState() & ~Qt::WindowMinimized);
+                        mainWindow.show();
+                        mainWindow.raise();
+                        mainWindow.activateWindow();
+                    });
     QObject::connect(&app, SIGNAL(openFile(QString)), &mainWindow, SLOT(openDatabase(QString)));
```

Comment:
Actually there is a `return 0` at line 61 (if another instance is already running).

(Maybe should that be return 1 instead?)